### ISSUE TITLE
Depend on dokku-docker-options instead of dokku-link

### ISSUE THREAD

**Issue by AlJohri** [jezdez/dokku-redis-plugin#3]
The `dokku-link` plugin suggests using the https://github.com/dyson/dokku-docker-options plugin instead which wraps all docker args. See this [issue](https://github.com/rlaneve/dokku-link/issues/5).

Would it be possible to depend on on `dokku-docker-options` instead of the `dokku-link` plugin?

**Comment by jezdez** [comment]
Yeah, why not, although I don't have time to work on a fix right now.
